alerta/auth/oidc.py

```diff
@@ -35,8 +35,8 @@ def get_oidc_configuration(app):
     except Exception as e:
         raise ApiError('Could not get OpenID configuration from well known URL: {}'.format(str(e)), 503)
 
-    if config['issuer'] != issuer_url:
-        raise ApiError('Issuer Claim does not match Issuer URL used to retrieve OpenID configuration', 503)
+    #if config['issuer'] != issuer_url:
```

Comment:
We need to understand why this verification of the issuer fails rather than just comment it out.

Reply:
See comment above about extra slash. This should be uncommented.